packages/edge-login-ui-rn/src/common/constants/Theme.js

```diff
@@ -3,6 +3,7 @@
 import { Platform } from 'react-native'
 
 export const HEADER_HEIGHT = Platform.OS === 'ios' ? 42 : 60
+export const HEADER_HEIGHT_1 = Platform.OS === 'ios' ? 46 : 66
```

Comment:
Where is this used?

Reply:
On the headers, somehow to have a uniform spacing I used that on the create account headers. I don't want to change a default that may be used by other components

Reply:
Ok my bad, I thought I did use it somehow, forgot to remove it, thanks.